scvi/model/base/_base_model.py

```diff
@@ -773,6 +779,62 @@ def convert_legacy_save(
             **save_kwargs,
         )
 
+    @classmethod
+    def convert_legacy_mudata_save(
+        cls,
+        dir_path: str,
+        output_dir_path: str,
+        overwrite: bool = False,
+        prefix: str | None = None,
+        **save_kwargs,
+    ) -> None:
+        """Converts a legacy MuData-based model (<1.XX) to the updated save format.
```

Comment:
Leaving 1.XX as a placeholder pending discussion on when this change should be incorporated